static/js/portico/dev-login.js

```diff
@@ -5,12 +5,12 @@ $(() => {
     // dev_login.html is rendered.
     if ($("[data-page-id='dev-login']").length > 0) {
         if (window.location.hash.substring(0, 1) === "#") {
-            /* We append the location.hash to the formaction so that URL can be
+            /* We append the location.hash to the next so that URL can be
```

Comment:
"to the next" reads funny here.  I would say something more like "to the input field with class `next`".

Reply:
Thanks, Showell for your review. I am new here and will love to learn from a contributor.